tensorflow_addons/callbacks/average_model_checkpoint.py

```diff
@@ -78,20 +78,20 @@ def set_model(self, model):
                 "with MovingAverage or StochasticAverage"
             )
 
-    def _save_model(self, epoch, logs):
+    def _save_model(self, **kwargs):
```

Comment:
Should we change this to the following in order to also support positional arguments:
```suggestion
    def _save_model(self, *args, **kwargs):
```
This seems to be more closely match the [signature used in the keras code](https://github.com/keras-team/keras/blob/83ac1ed7f8293c1921996a7ddd61fe7e8e66c5ca/keras/callbacks.py#L1395).